dockerfiles/Dockerfile.arm32v7

```diff
@@ -1,6 +1,24 @@
-FROM arm32v7/ubuntu:16.04
-
-RUN apt-get update && apt-get install -y sudo build-essential curl libcurl4-openssl-dev libssl-dev wget python3 python3-pip python3-dev git
+FROM balenalib/raspberrypi3-python:latest-stretch-build
```

Comment:
What is balenalib/raspberrypi3-python? Where to get it?

Reply:
It is a Raspbian Stretch base image frequently used to build Raspberry Pi bits. It comes with built-in support for cross-compile and some other goodies.

It is published by https://www.balena.io/, and the image itself is automatically pulled from Docker Hub: https://hub.docker.com/u/balena/ (the user does not need to get it themselves).